### PR TITLE
Styled 2

### PR DESCRIPTION
**Pull request by fybits**
Now with resolved conflict.
Styles for tables, active links and forms for creating and editing members, tests and member-tests.

### CODE REVIEW

File: app/views/layouts/pdf.html.erb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 <title>Levi Tests</title>
+    <%# <%= wicked_pdf_stylesheet_link_tag "https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600&display=swap" %> %>
```
Comment (fybits) [inline app/views/layouts/pdf.html.erb:5]: Не могу подключить шрифт в pdf

File: app/views/members/_form.slim [hunk 2]
```diff
@@ -3,5 +3,5 @@
     = f.input :name
     = f.input :email
   = f.actions
-    = f.action :submit, as: :button, button_html: { class: 'btn btn-primary' }
-    = f.action :cancel, as: :link, button_html: { class: 'btn btn-secondaty' }
\ No newline at end of file
+    = f.action :cancel, as: :link, button_html: { class: 'btn btn--secondary wide' }
+    = f.action :submit, as: :button, button_html: { class: 'btn btn--primary wide' }
```
Comment (fybits) [inline app/views/members/_form.slim:7]: Unused wide classes

File: app/assets/stylesheets/footer.scss [hunk 3]
```diff
@@ -1,8 +1,8 @@
 @import "./variables.scss";
 
 .footer {
-  position: absolute;
-  bottom: 0;
+  // position: sticky;
```
Comment (DmitryLeviossa) [inline app/assets/stylesheets/footer.scss:4]: Remove commented code

File: app/views/members/_form.slim (hunk 2 above)
Comment (DmitryLeviossa) [reply app/views/members/_form.slim:7]: ?

File: app/views/questions/_form.slim [hunk 4]
```diff
@@ -1,14 +1,18 @@
 = semantic_form_for question do |f|
-  = question.errors.full_messages
+  / = question.errors.full_messages
```
Comment (DmitryLeviossa) [inline app/views/questions/_form.slim:2]: Remove commented code

**Pull request merged**